src/index.md

```diff
@@ -8,6 +8,93 @@ styles:
 
 <div class="content-sizing" markdown="1">
 
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+build web apps that work offline.
+
+Modern offline web apps are possible thanks to **service workers** (SWs). SWs
+are background workers, written in JavaScript, that can locally save all of
+the resources needed to run your site offline. The SW also serves as the
+network proxy for your web app. In other words, in your SW code, you specify
+whether a resouce should be served from the device's local storage (AKA the
+**cache**), or from the network. Or perhaps the app should try to get a
+fresh copy of the resource from the network first, and then fall back
+to the version in the cache if the network is taking too long, and so on.
+
+The logic for determining whether to fetch resources from the cache or
+from the network is called the service worker's **caching strategy**. There
+are a lot of strategies, and which one you use depends on the resource you're
+caching. For example, you can probably serve a logo that doesn't change often
+straight from the cache. But for a JSON file containing a user's social media
+feed, you probably want to display the cached content only while you fetch the
+latest content from the network.
+
+Caching strategy and implementation is one of the trickiest parts of
+building offline web apps. *The "core" Workbox libraries and tools*
+(`workbox-sw`, `workbox-build`, `workbox-cli`) *simplify this process.*
+
+Note: these tools and libraries are only "core" in the sense that they are
+the most popular Workbox products. Workbox also has libraries for other features,
+such as saving Google Analytics activity that occurs while a user is offline, and then
+sending that data up when the device goes back online. You can use these other
+libraries independently of `workbox-build`, etc. They don't depend on the "core" tools
+and libraries. This overview won't be covering any of these other Workbox products.
+
+`workbox-build` and `workbox-cli` generate a complete service worker for
+you at build time. By default, they generate caching strategies that work
+well for most simple apps, but you can also customize the strategies to meet
+your particular needs. That's the first main feature.
+
+The second is tracking
+whether there have been any changes to the resources that you need to save
+for offline. When a service worker first gets installed, it saves the
+current state of all the resources that you specify. But what happens
+when you update a resource after this initial installation? How do you
+get the service worker to fetch the updated version? The Workbox
+tools automate this process by tracking changes to your resources and
+ensuring that the resources get updated the next time that the app is online.
+
+If you've got a simple site and you just want it to work offline, then
```

Comment:
I might add a section header here, it wasn't entirely sure that you've moved from generating a full service worker with workbox-build or workbox-cli to injecting the precaching into a custom sw.